hack/.staticcheck_failures

```diff
@@ -244,6 +244,5 @@ vendor/k8s.io/kubectl/pkg/cmd/wait
 vendor/k8s.io/kubectl/pkg/describe/versioned
 vendor/k8s.io/kubectl/pkg/scale
 vendor/k8s.io/legacy-cloud-providers/aws
-vendor/k8s.io/legacy-cloud-providers/azure
```

Comment:
why remove this line? any tool you are using to remove this?

Reply:
It is ref from [here](https://github.com/kubernetes/kubernetes/issues/81657#issue-482850808), all the changes fix the staticcheck in this package, remove this line indicate that this package is free from staticcheck.